content/en/04-tutorials/04-custom-data/01-distributed-bpftrace-deployment.md

```diff
@@ -170,30 +170,37 @@ If you'd like to filter the results to a particular service, modify line 67 to i
 df = df[px.contains(df['dst'], 'sock-shop')]
 ```
 
+### Tracepoint status
+
+Run `px/tracepoint_status` to see the information about all of the tracepoints running on your cluster. The `STATUS` column can be used to debug why a tracepoint fails to deploy. 
+
 ### Running other bpftrace programs
 
-With a few modifications to obey the Rules listed above, all of the TCP [bpftrace](https://github.com/iovisor/bpftrace) programs are known to work with Pixie:
+The following [bpftrace](https://github.com/iovisor/bpftrace) programs are available today for use in Pixie:
+
+- `capable.bt`: use the `pxbeta/capable` script
+- `dcsnoop.bt`: use the `pxbeta/dc_snoop` script. 
+- `mdflush.bt`: use the `pxbeta/md_flush` script. 
+- `naptime.bt`: use the `pxbeta/nap_time` script. 
+- `oomkill.bt`: use the `pxbeta/oom_kill` script. 
+- `syncsnoop.bt`: use the `pxbeta/sync_snoop` script. 
+- `tcpdrop.bt`: use the `px/tcp_drops` script. 
+- `tcpretrans.bt`: use the `px/tcp_retransmits` script. 
+
+With a few modifications to obey the Rules listed above, all of the TCP bpftrace programs are known to work with Pixie:
```

Comment:
1) Rules -> rules.

2) Instead of listing the ones below, can we just something more generic:

"Many other bpftrace programs can work with Pixie. Some may require a few modifications to obey the rules listed above."